python/oneflow/one_embedding.py

```diff
@@ -303,12 +457,66 @@ def make_cached_host_mem_store_options(
 
 
 def make_uniform_initializer(low, high):
+    """make uniform initializer param of make_table
+
+    Args:
+        low (float): A python scalar. Lower bound of the range of random values to generate.
+        high (float): A python scalar. Upper bound of the range of random values to generate.
+
+    Returns:
+        dict: initializer param of make_table
+    
+    For example:
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> initializer = flow.one_embedding.make_uniform_initializer(low=-scale, high=scale)
+        >>> # pass the initializer to flow.one_embedding.make_table
```

Comment:
```suggestion
        >>> # pass the initializer to flow.one_embedding.make_table
        >>> # ...
```

感觉是不是这类注释表示余下步骤的地方，多输入一行 `...` 比较好。其它地方也是类似的。

Reply:
已改